src/test/java/com/musicshare/playlistapi/controller/PlayListControllerTest.java

```diff
@@ -30,4 +34,24 @@ public void test_createPlayListWithName() throws Exception {
                 .andExpect(jsonPath("$.songs", hasSize(0)));
 
     }
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    /**
+     * Add the song to the existing playlist.
+     * @throws Exception
+     */
+    @Test
+    public void test_addSongsToPlaylist() throws Exception {
+        test_createPlayListWithName();
```

Comment:
create a playlist object and add it instead of using the other test

Reply:
Implemented the review comments